notebooks/Customer_Automotive_Churn_Predictions.ipynb

```diff
@@ -0,0 +1,1866 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Customer_Automotive_Churn_Predictions: PyMapD- XGBoost - MapD\n",
+    "# Response Variable: Col_1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Importing Libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pymapd\n",
+    "import pygdf\n",
+    "import time\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from copy import deepcopy\n",
+    "from sklearn.preprocessing import LabelEncoder\n",
+    "le= LabelEncoder()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data Extaction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setting up MapD connection"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Connection(mapd://mapd:***@http://localhost:9090/mapd?protocol=http)\n"
+     ]
+    }
+   ],
+   "source": [
+    "dbname    = 'mapd'\n",
+    "username  = 'mapd'\n",
+    "password  = 'HyperInteractive'\n",
+    "hostname  = 'localhost'\n",
```

Comment:
Might want to have a note saying to change this to `mapd-core` if running via `nvidia-docker-compose`